OmniBLE/OmnipodCommon/BeepType.swift

```diff
@@ -9,27 +9,19 @@
 
 import Foundation
 
-// BeepType is used for the $19 Configure Alerts and $1F Cancel Commands
-// Values 1 thru 8 are exactly the same as in BeepConfigType below
-// N.B. for BeepType, noBeep is 0x0, while for BeepConfigType it is 0xF
+//
+// BeepType is used for the 0x19 Configure Alerts, 0x1E Set Beep Options and 0x1F Cancel Delivery commands.
+// Some beep types values behave differently based on the command & circumstances due to Omnipod internals.
```

Comment:
nit: Maybe making separate enums for the different commands would make it clearer, and allow you specify which values are allowed for which commands using strong typing.  `noBeepCancel` is a confusing name, for example.

Reply:
Understood, In this case I found while prototyping some new alert beep control code that using separate enums per command for the same underlying entity made things less clear in places and didn’t really help the value naming. The ConfigureAlerts command needs two different “no beep” values, one when cancelling an existing alert & one for creating a no beep pod alert, and noBeepCancel is suitable both for a canceling ConfigureAlerts command as well as for a no beep CancelDelivery command.